astronomer/providers/apache/hive/sensors/named_hive_partition.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Dict, Optional
+
+from airflow.exceptions import AirflowException
+from airflow.providers.apache.hive.sensors.named_hive_partition import (
+    NamedHivePartitionSensor,
+)
+
+from astronomer.providers.apache.hive.triggers.named_hive_partition import (
+    NamedHivePartitionTrigger,
+)
+
+
+class NamedHivePartitionSensorAsync(NamedHivePartitionSensor):
+    """
+    Waits for a set of partitions to show up in Hive.
+
+    :param partition_names: List of fully qualified names of the
+        partitions to wait for. A fully qualified name is of the
+        form ``schema.table/pk1=pv1/pk2=pv2``, for example,
+        default.users/ds=2016-01-01. This is passed as is to the metastore
+        Thrift client ``get_partitions_by_name`` method. Note that
+        you cannot use logical or comparison operators as in
+        HivePartitionSensor.
+    :param metastore_conn_id: Reference to the
+        :ref:`metastore thrift service connection id <howto/connection:hive_metastore>`.
```

Comment:
we don't have a howto guide in our docs, so those won't link to anything

Reply:
Agree, removed it.